package.json

```diff
@@ -32,6 +32,7 @@
     "passport-local": "^1.0.0",
     "serialport": "^4.0.7",
     "sqlite3": "^3.1.8",
+    "uuid": "^3.1.0",
```

Comment:
This change is probably needed for something else but seems unrelated to this PR